charts/pulsar/templates/log4j-configmap.yaml

```diff
@@ -0,0 +1,143 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.broker.component }}-log4j"
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "pulsar.standardLabels" . | nindent 4 }}
+    component: {{ .Values.broker.component }}
+data:
+  # Include prometheus configuration file, setup to monitor all the
+  # Kubernetes pods with the "scrape=true" annotation.
+  log4j2.yaml: |
```

Comment:
I don't think we need to add a separate config map for each component. We should just add the `log4j2.yaml` field to the existing configmap for each component.

Reply:
I don't think this will have any impact. From the user's point of view, I think it will be more friendly. Do we provide a unified configuration place for the log, or do we disassemble it to configure it. I prefer the former.

Reply:
I don't think this will have any impact. From the user's point of view, I think it will be more friendly. Do we provide a unified configuration place for the log, or do we disassemble it to configure it. I prefer the former.